Update script.py

ANKIT9761 opened the pull request:
No need to add Course name in script you have to only enter your icloud id and password.Removed the search functionality as we can skip it the website by selecting the course and then clicking next.

ANKIT9761 committed:
Update script.py

rahulchauhan049 commented:
I like your Idea of skipping the search part. 
There are some merge conflicts, please resolve them so I can merge. Make sure you are up to date with the main branch.

rahulchauhan049 commented on script.py:
```diff
@@ -1,73 +1,67 @@
 from selenium.webdriver.support.select import Select
 
-driver = webdriver.Chrome()
+driver = webdriver.Chrome('D:/webdriver/chromedriver.exe')
```
This will not work in other Os. You can remove this path

ANKIT9761 replied:
ok

rahulchauhan049 commented on script.py:
```diff
@@ -1,73 +1,67 @@
-password.send_keys('password')  ##################################### Change This Text ############################
+# Change user name and passwords with your icloud username and password
+username.send_keys('17SCSE112003')  ##################################### Change This Text ############################
+password.send_keys('Gourav8211@')  ##################################### Change This Text ############################
```
You should remove this username and password and replace them with a placeholder.

rahulchauhan049 commented on script.py:
```diff
@@ -1,73 +1,67 @@
+time.sleep(10)
+#search = driver.find_element_by_class_name('fa-search')
+# search = driver.find_element_by_xpath('/html/body/div[1]/div/div[1]/div[3]/div/div[2]/div/form/div/div[3]/button')
+#search.click()
```
Can you please remove this commented code?

rahulchauhan049 commented on script.py:
```diff
@@ -1,73 +1,67 @@
-time.sleep(2)
 
 start = driver.find_element_by_class_name('btn-rounded')
+#start = driver.find_element_by_xpath('/html/body/div[1]/div[1]/div/div[3]/div[2]/div[2]/center/input')
```
Remove this commented code also

rahulchauhan049 commented on script.py:
```diff
@@ -1,73 +1,67 @@
 
-time.sleep(4)
-
+time.sleep(2)
```
Increase time to 5 seconds. This page take more time to load on slow internt

ANKIT9761 replied:
I have done all the changes thanks for pointing them out.

ANKIT9761 committed:
Merge branch 'main' into main

viditvarshney commented on script.py:
```diff
@@ -86,14 +75,14 @@
         for suggestion_box in suggestion_boxes:
             try:
-                suggestion_box.send_keys(' Teacher')
+                suggestion_box.send_keys('Teacher')
```
Give Space Before "Teacher" 
the comment will be: " Good Teacher

ANKIT9761 replied:
Noted

ANKIT9761 committed:
Update script.py

ANKIT9761 reviewed (commented):
Suggestions box content is updated as requested

ANKIT9761 committed:
Update script.py

ANKIT9761 committed:
Update script.py

I think the class selectors are giving a little bit of problem as different specialization seems to use different classes for button.I have tested this and i think getting the button by x path method is most suitable here.

ANKIT9761 committed:
Update README.md

Updated the readme as we no longer needed to search the course name and update it manually the script will automatically select the course which you are Enrolled in.
